server/app/services/program/ProgramService.java

```diff
@@ -251,12 +272,10 @@ private CompletionStage<ProgramDefinition> syncProgramAssociations(ProgramModel
         syncProgramDefinitionQuestions(
             programRepository.getProgramDefinition(program), maxVersionForProgram);
 
```

Comment:
**note:** We Remove the getQuestionCacheEnabled() check here because we check if it is enabled as part of the setFullProgramDefinitionCache method.